operator/controllers/keptnworkload_controller.go

```diff
@@ -19,26 +19,28 @@ package controllers
 import (
```

Comment:
since all other controllers are in their own directories now, let's make a directory for this little guy as well